core/src/main/java/org/openstack4j/openstack/compute/internal/ServicesServiceImpl.java

```diff
@@ -78,4 +79,38 @@ public ExtService disableService(String binary, String host) {
 
 		return put(ExtService.class, uri("/os-services/disable")).entity(ServiceAction.disable(binary, host)).execute();
 	}
-}
\ No newline at end of file
+
+    /**
+     * Forcefully shuts down a compute service.
+     * <p>Author:Jyothi Saroja Daggu</p>
+     * @Title: forceDownService
+     * @param binary
+     * @param host
+     * @return
+     * @see org.openstack4j.api.compute.ServicesService#forceDownService(java.lang.String, java.lang.String)
```

Comment:
It is pointless to add tags that have no other values apart from what is obvious from the signature, like: `@param binary` or empty `@return`. Also, references to interface method the documented method is overriding is redundant.

Reply:
Removed the tags